### ISSUE TITLE
Wem 7536745 was expected to start at offset 41 but instead started at offset 141559880  Same issue as #19

### ISSUE THREAD

**Issue by ghramsey** [hpxro7/bnkutil#31]
I have this exact same error as #19  except the file is for Alien Isolaton:

![image](https://github.com/user-attachments/assets/892e8c02-bb85-4321-93c5-74694c05e4d3)

**Comment by marcosdotme** [comment]
@ghramsey Have you found a way to replace WEM inside BNK files? I found a way to replace WEM inside PCK, but BNK didnt work... I tried to use at least 5 different tools @_@

**Comment by ghramsey** [comment]
@marcosdotme You mean for Alien Isolation?    I actually didn't find anything I wanted or needed to edit inside te BNK
files so I have not tried them.  The PCK files I did end up making work and retaining the subtitle information by setting the
correct EventID insde the WEM file.

I ended up using a program called FusionTools which is available on NexusMods..
It's for editing another CA game' Warhammer , I think

**Comment by marcosdotme** [comment]
@ghramsey Yes, I mean for Alien Isolation, sorry. I used FusionTools and Wwise 2013 for PCK. I did the same step-by-step for BNK file but it don't work, so I'm trying to figure it out if I'm doing something wrong or if its impossible to edit BNK lol
